Backend/Lyzer/obj/Debug/net8.0/Lyzer.sourcelink.json

```diff
@@ -0,0 +1 @@
+{"documents":{"C:\\Users\\Benge\\OneDrive\\Documents\\School\\Lyzer\\*":"https://raw.githubusercontent.com/LittleClumsy/Lyzer/c125a28144ff928f27e9465431b12f96288a3733/*"}}
```

Comment:
Same here, could we please remove and add to `.gitignore`

`./Backend/Lyzer/obj/`